packages/loader/driver-utils/src/insecureUrlResolver.ts

```diff
@@ -28,6 +28,7 @@ export class InsecureUrlResolver implements IUrlResolver {
 	private readonly cache = new Map<string, Promise<IResolvedUrl>>();
 
 	constructor(
+		private readonly deltaStreamUrl: string,
```

Comment:
Nit: let's put this after storageUrl, so the first one remains the "most general" one. The call sites of the constructor won't immediately complain but let's make sure to reorder them there as appropriate.

Reply:
Oh ok, thought they were doing some kind of alphabetical order